test/gleam/string_test.gleam

```diff
@@ -38,6 +38,20 @@ pub fn split_test() {
   |> should.equal(["Gleam", "Erlang,Elixir"])
 }
 
+pub fn split_first_test() {
```

Comment:
`split_once_test`?

Nice, I was also interested in adding this function but wondering what to call it. Ik believe it's `string.break` in Haskell (when a tuple is returned instead of a list). But I think either `split_once` or `split_first` also sounds right.

Reply:
ReasonML has `bounded_split` which returns a list and a parameter that designates to split into at most n substrings.

Reply:
But `bounded_split` takes a regex: https://reasonml.github.io/api/Str.html
I was also considering to add it to: https://github.com/gleam-lang/stdlib/pull/60

Naming is hard...

Reply:
Good spot, have updated the name.

Naming is hard. break sounds alright but break vs split could probably just as easily be applied the opposite way round